tests/raw_query/tests.py

```diff
@@ -229,7 +229,7 @@ def test_many_to_many(self):
 
     def test_extra_conversions(self):
         """
-        Test to insure that extra translations are ignored.
```

Comment:
You can use the single line style if it fits: `"""Extra translations are ignored."""`